erik/calculator/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Sarabun:wght@100&family=Work+Sans:wght@300;360&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="main_div">
+      <div class="dots">
+        <div class="dot1"></div>
+        <div class="dot2"></div>
+        <div class="dot3"></div>
+      </div>
+      <div class="calcInput">
+        <input id="stach" hidden value="" />
+        <input id="output_value" type="text" placeholder="0" disabled />
+      </div>
+      <div class="first_row">
+        <div class="clear" type="button">C</div>
+        <!-- onclick="btnClear()" -->
+        <div class="pos_neg">+/-</div>
+        <!-- onclick="negPos()" -->
+        <div class="btn-operator precent" id="%">
+          %
+        </div>
+        <!-- onclick="percentFunc('%')" -->
+        <input type="button" class="btn-operator divide" value="/" />
+      </div>
+      <div class="second_row">
+        <input type="button" class="btn_numbers" value="7" />
```

Comment:
typically class names would be singular. Here for instance, with `btn_numbers` it would typically be seen as `btn_number`... unless the element had multiple numbers within it.